### ISSUE TITLE
Current Status Of Protocol

### ISSUE THREAD

**Issue by jmwright** [forgefed/forgefed#24]
I was wondering what the current status of this project is. I see that there is a beta specification [here](https://browse.git.joinout.de/forgefed.git/blob/master/forgefed.md), but I don't see activity after Aug-Sept 2018. Is work still ongoing? Are there any reference implementations?

**Comment by bill-auger** [comment]
fr33domlover posted an update just today to the ActivityPub forum - and
also suggested starting a similar web forum on that same host

https://socialhub.network/t/forgefed-project-hosting-federation/341

**Comment by jmwright** [comment]
Great, thanks!

**Comment by bill-auger** [comment]
ill re-open for a few days in case others want to add something in response without signing up for that other website

**Comment by genofire** [comment]
Maybe at this to README

**Comment by bill-auger** [comment]
## ForgeFed Community Forum

In order to consolidate development and community discussions, there is now a public web forum on the socialhub.network website. Everyone is invited to use that forum instead of this GitHub repo or the mailing list.
Thanks to SocialHub and FeNeAs for making that possible.

Web Forum: https://socialhub.network/c/forgefed


## ForgeFed in the Fediverse

There is also a user on the Mastodon network to which you can subscribe for progress updates.

Fediverse: `@forgefed@floss.social` (https://floss.social/@forgefed)


## ForgeFed Repository

This GitHub repo has not been updated in some time, and it is unclear at the moment whether or not @yookoala still wants to maintain it. The NotABug repo contains the most recent documentation and reference source code, all currently under the maximally permissive CC0 license.

Repository: https://notabug.org/peers/forgefed

**Comment by 6543** [comment]
ad least add the new repo to the github repo description

**Comment by 6543** [comment]
-> https://codeberg.org/ForgeFed/forgefed/issues/101#issuecomment-458010
-> https://notabug.org/peers/forgefed/pulls/103

**Comment by 6543** [comment]
more:
https://todon.nl/@fr33domlover/107579920154669382
https://chaos.social/web/@6543/108222466960369092

**Comment by bill-auger** [comment]
> ad least add the new repo to the github repo description

it is not possible - the only thing that anyone can do is to moderate the issues

**Comment by yookoala** [comment]
@bill-auger: All people in the workgroup has write permission to this repository. I believe you can change the README to this repository. If not, please let me know how better to setup the permission. Thanks.

**Comment by bill-auger** [comment]
@yookoala IIRC we tried to modify the VCS it was not possible - someone opened a pull request and it still open, because no one could accept it

wow glad yer back though :) - thanks - there are still many links floating around the web that point here; and it always the same confusion, trying to tell people why nothing can change

this would be the simplest thing you could to do:

go to https://github.com/orgs/forgefed/people

you will see all "people" are "member"; and only you are "owner" - members can only moderate the tickets, and create new repos, not much else

press the gear icon beneath the "Role" column for each person and press "Change Role", then select "Owner" - there is also "Admin" - owner has the most privileges - im not sure what is the difference - if you do not plan to be around, it would be best to make a few people "Owner", to be sure that someone has the maximum abilities

**Comment by yookoala** [comment]
@bill-auger Sorry. Didn't realize that.

Add you with Admin permission. Please check.

**Comment by bill-auger** [comment]
i was able to push to the VCS - thanks - it would be nice to have the organization logo too

**Comment by bill-auger** [comment]
yes - that did the trick - i was able to set the organization's description, and the "invite members" button is available - thanks again :)

**Comment by yookoala** [comment]
@bill-auger: Thanks for the effort. Should have done this earlier.

**Comment by 6543** [comment]
@bill-auger https://codeberg.org/ForgeFed/ForgeFed/issues/95#issuecomment-469635

I would like to make this repo as a push mirror (same as what NAB repo is now)

**Comment by 6543** [comment]
done - now it get synced every 2h
